src/common/kyc/index.js

```diff
@@ -235,7 +243,7 @@ const getSession = async (config, authenticate, dispatch, hardwareWalletType) =>
 			}
 			await session.establish();
 		} catch (error) {
-			console.log(error);
+			log.error('getSession trezor %s', error);
```

Comment:
@sk91 it could be ledger too. I suggest changing to hardware wallet instead of trezor. cheers

Reply:
fixed